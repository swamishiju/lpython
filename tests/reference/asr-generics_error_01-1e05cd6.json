{
    "basename": "asr-generics_error_01-1e05cd6",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/generics_error_01.py",
    "infile_hash": "2c26a040a0d03561f54a6af9cdd9db262fa0e9db7a6db2b115f6c4eb",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_error_01-1e05cd6.stderr",
    "stderr_hash": "c906edc41f270197a279f3bcf5bf514ba66eebb7b10f53cefefd8b2a",
    "returncode": 2
}