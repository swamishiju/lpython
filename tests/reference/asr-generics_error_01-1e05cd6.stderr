semantic error: Type mismatch in procedure call; the types must be compatible
  --> tests/errors/generics_error_01.py:12:9
   |
12 | print(f(1,"a"))
   |         ^ type mismatch (passed argument type is i32 but required type is T)
   |
 9 | def f(x: T, y: T) -> T:
   |          ^ type mismatch (passed argument type is i32 but required type is T)
